components/audio-player.js

```diff
@@ -0,0 +1,15 @@
+import ReactAudioPlayer from 'react-audio-player'
+
+const AudioPlayer = ({
+  url
+}) => (
+  <ReactAudioPlayer
```

Comment:
Seems fine to use this library but is there any advantage in using it over the audio tag? Seems like mostly it’s providing functions/props for events but we’re not needing them: https://github.com/justinmc/react-audio-player/blob/master/src/index.tsx#L158